src/main/java/org/alliancegenome/curation_api/interfaces/crud/GeneExpressionAnnotationCrudInterface.java

```diff
@@ -0,0 +1,43 @@
+package org.alliancegenome.curation_api.interfaces.crud;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import org.alliancegenome.curation_api.interfaces.base.BaseIdCrudInterface;
+import org.alliancegenome.curation_api.model.entities.GeneExpressionAnnotation;
+import org.alliancegenome.curation_api.model.ingest.dto.fms.GeneExpressionFmsDTO;
+import org.alliancegenome.curation_api.response.APIResponse;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.response.SearchResponse;
+import org.alliancegenome.curation_api.view.View;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import java.util.HashMap;
+import java.util.List;
+
+@Path("/expression-annotation")
+@Tag(name = "CRUD - Expression Annotations")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface GeneExpressionAnnotationCrudInterface extends BaseIdCrudInterface<GeneExpressionAnnotation> {
+
+	@GET
+	@Path("/findBy/{identifier}")
+	@JsonView(View.FieldsAndLists.class)
+	ObjectResponse<GeneExpressionAnnotation> getByIdentifier(@PathParam("identifier") String identifier);
+
+	@Override
```

Comment:
Overriding this endpoint isn't doing anything useful - the reason we've overridden search endpoints in other classes is because they are using a different view

Reply:
You've deleted the wrong endpoint here. The /`findBy/{identifier}` endpoint needs to be put back in.  It is the overriding of the` /search` endpoint that is unnecessary

Reply:
fixed